fe1-web/src/features/rollCall/screens/__tests__/ViewSingleRollCall.test.tsx

```diff
@@ -135,6 +143,30 @@ describe('EventRollCall', () => {
         're-opened roll calls',
         testRender(mockRollCallReopened, false, ['attendee1', 'attendee2']),
       );
+      it('while closing roll call', () => {
```

Comment:
name not really explicit of what it is testing

Reply:
The name is concatenated with the above `describe`. Wich leads to `no duplicate attendees while closing roll call`.